x/campaign/types/genesis_test.go

```diff
@@ -140,11 +138,13 @@ func TestGenesisState_Validate(t *testing.T) {
 				},
 				CampaignCounter: 2,
 				MainnetAccountList: []types.MainnetAccount{
-					sample.MainnetAccount(330, sample.Address()),
-					sample.MainnetAccount(434, sample.Address()),
+					{
+						CampaignID: 330,
+						Address:    "330",
+					},
```

Comment:
```suggestion
					sample.MainnetAccount(330, sample.Address()),
```

Reply:
If `sample.Address()` is used I cannot assert exact error message which contain the exact address. I will use `sample.MainnetAccount()` but without `simple.Address()`

Reply:
you can create the address in the top level of the function and reuse it in the test and the log